addonscript/requirements/base.py37.txt

```diff
@@ -313,28 +313,27 @@ rsa==4.7.2 \
     --hash=sha256:78f9a9bf4e7be0c5ded4583326e7461e3a3c5aae24073648b4bdfa797d78c9d2 \
     --hash=sha256:9d689e6ca1b3038bc82bf8d23e944b6b6037bc02301a574935b2dd946e0353b9
     # via python-jose
-scriptworker==38.2.0 \
-    --hash=sha256:142ce7e7323c2de73d947896540717db8fad6973030fde7a3b047f846466cede \
-    --hash=sha256:bcd14a35a6d8affdb7d29e63125bc7966c32fc42daba387a50566238bd25a8e8
+scriptworker==39.0.0 \
+    --hash=sha256:5b5a0a3caffcaf833ed615ea6b65cf147da373721fbb5054cc30fca8d9f3a1d1 \
+    --hash=sha256:b6097f8e2267f6d0925b3453d1f982fc9972ddec3926cd666808346cfd95c479
     # via -r requirements/base.in
 six==1.16.0 \
     --hash=sha256:1e61c37477a1626458e36f7b1d82aa5c9b094fa4802892072e49de9c60c4c926 \
     --hash=sha256:8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254
     # via
     #   ecdsa
     #   jsonschema
-    #   jwcrypto
     #   mohawk
     #   python-dateutil
     #   taskcluster
 slugid==2.0.0 \
     --hash=sha256:a950d98b72691178bdd4d6c52743c4a2aa039207cf7a97d71060a111ff9ba297 \
     --hash=sha256:aec8b0e01c4ad32e38e12d609eab3ec912fd129aaf6b2ded0199b56a5f8fd67c
     # via taskcluster
-taskcluster==39.2.0 \
-    --hash=sha256:07dec0b12c80122498d772a242f92373d65731f133fb0d9aba5c2adf78475b13 \
-    --hash=sha256:844d58207f7ccba73cce2413ff79ad83ef39e4095e0269fb82ff14fa9b1c3e2f \
-    --hash=sha256:e5fe89c1f85b3caf141106053c1e98bcb930a629889e67acb2bd9aa92bd95182
+taskcluster==44.1.0 \
```

Comment:
Hmmmm, scriptworker integration tests are breaking, might be worth more investigation =\